src/models/booking.cr

```diff
@@ -59,6 +59,7 @@ class Booking
     booking_model.user_id = booking_model.booked_by_id if !booking_model.user_id_column.defined?
     booking_model.booked_by_email = booking_model.booked_by_email.downcase
     booking_model.user_email = booking_model.booked_by_email if !booking_model.user_email_column.defined?
+    booking_model.user_email = booking_model.user_email.downcase
```

Comment:
IIRC, you did some work on emails and email digests to preserve case sensitivity. Is there a justification for the general downcasing here?